### DISCUSSION TITLE
[WASM] I'm on 5.0.0-dev-XXX already. 4.10 is out as of today. Now what?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#13558] [Q&A] [answered]
I'm on 5.0.0-dev-XXX already since I wanted to get fixes from @TopProgrammer77 in my work. Now 4.10 is out as official release.

What would I need to do?
- wait for first official 5.X?
- switch back to 4.10 - and still get the fixes from @TopProgrammer77?
- else?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the feedback. All fixes that are in 4.10 originate from 5.0-dev (which is still being worked on), so if you're on 5.0-dev, you're already current.

**Comment by DierkDroth** [comment]
Thanks @jeromelaban. I'll then stay on 5.0.0-dev-XXX
